app/templates/teachers/edit-student-modal.jade

```diff
@@ -6,8 +6,38 @@ block modal-header-content
     span=view.user.broadName()
     
 block modal-body-content
-        
+
+  - var unusedEnrollments = view.prepaids.totalMaxRedeemers() - view.prepaids.totalRedeemers()
+  - var noLicenses = unusedEnrollments == 0
```

Comment:
Can instead use totalAvailable method from app/collections/Prepaids here- https://github.com/codecombat/codecombat/blob/master/app/collections/Prepaids.coffee#L26
Like view.prepaids.totalAvailable() - can also directly use it on line 21